CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### To release
+
+  * Symfony 5 compatibility added
+  * Drop tests for PHP 5.4
```

Comment:
Dropping tests for PHP 5.4 without dropping support for it in composer.json is a bad idea IMO. Projects running on PHP 5.4 should rather get an old test release than an uptodate release for which no testing has been done (as it might be incompatible).